python/paddle/tensor/creation.py

```diff
@@ -1923,3 +1923,88 @@ def tril_indices(row, col, offset=0, dtype='int64'):
                              'dtype': dtype
                          })
     return out
+
+
+def triu_indices(row, col=None, offset=0, dtype='int64'):
```

Comment:
the title of [rfc](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20220813_api_design_for_triu_indices.md) should be `paddle.triu_indices` instead of `paddle.tril_indices`?

Reply:
> the title of [rfc](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20220813_api_design_for_triu_indices.md) should be `paddle.triu_indices` instead of `paddle.tril_indices`?

不好意思 我写错了。已重新提交rfc的PR https://github.com/PaddlePaddle/community/pull/224